projects/storefrontstyles/scss/components/myaccount/_my-coupons.scss

```diff
@@ -0,0 +1,55 @@
+%cx-my-coupons {
+  .cx-coupon-deck {
+    padding: 1.25rem 0 0 0;
+  }
+
+  .cx-coupon-spinner {
+    padding: 5rem 0 5rem 0;
+  }
+
+  .cx-coupon-card {
+    padding-bottom: 30px;
+  }
+
+  .cx-my-coupons-form-group {
+    padding: var(--cx-padding, 0);
+    margin-bottom: var(--cx-margin, 0);
+  }
+
+  .cx-my-coupons-sort {
+    display: flex;
+    justify-content: space-between;
+    margin: var(--cx-margin, 0);
+
+    @include media-breakpoint-down(sm) {
+      flex-direction: column;
+      padding-top: var(--cx-padding, 0);
+    }
+    &.top {
+      padding: var(--cx-padding, 1rem 0);
+      border-width: 0 0 1px 0;
+      border-style: solid;
+      border-color: var(--cx-color-light);
+    }
+
+    &.bottom {
+      padding: var(--cx-padding, 2rem 0 1rem 0);
+      border-width: 1px 0 0 0;
+      border-style: solid;
+      border-color: var(--cx-color-light);
+    }
+  }
+
+  .cx-mycoupon-thead-mobile {
+    @include media-breakpoint-down(sm) {
+      display: none;
+    }
+  }
+
+  .cx-my-coupons-notes {
+    a {
+      color: $dark;
+      font-weight: var(--cx-font-weight-bold);
```

Comment:
We're not using CSS variables for fonts anymore, we only use the `@include type()`
 **mixin** for fonts, you can use this reference table below:
![Screen Shot 2019-09-27 at 10 59 00 AM](https://user-images.githubusercontent.com/15113219/71752383-dee28200-2e4c-11ea-8d93-08dd27104fe8.png)

Reply:
updated to use    @include type('8');